Makefile

```diff
@@ -33,37 +17,8 @@ check-dictionary:
 sort-dictionary:
 	LANG=C sort ${SORT_ARGS} -u -o codespell_lib/data/dictionary.txt codespell_lib/data/dictionary.txt
 
-install: codespell manpage
-	install -d ${DESTDIR}${datadir} ${DESTDIR}${bindir} ${DESTDIR}${mandir}
-	install -m644 -t ${DESTDIR}${datadir} data/dictionary.txt data/linux-kernel.exclude
-	install -m755 -T codespell ${DESTDIR}${bindir}/codespell
-	install -d ${DESTDIR}${mandir}
-	install -m644 -t ${DESTDIR}${mandir} codespell.1.gz
-
-git-tag-release:
-	git commit -a -m "codespell $(VERSION)"
-	git tag -m "codespell $(VERSION)" -s v$(VERSION)
-	git gc --prune=0
-
-codespell-$(VERSION).tar.xz.asc: codespell-$(VERSION).tar.xz
-	gpg --armor --detach-sign --output $@ $^
-
-codespell-$(VERSION).tar.xz:
-	git archive --format=tar --prefix codespell-$(VERSION)/ v$(VERSION) | xz > $@
-
-tar-sync: codespell-$(VERSION).tar.xz codespell-$(VERSION).tar.xz.asc
-	github-release release --repo codespell --tag v$(VERSION) --name v$(VERSION)
-	github-release upload  --repo codespell --tag v$(VERSION) \
-		--name codespell-$(VERSION).tar.xz \
-		--file codespell-$(VERSION).tar.xz
-	github-release upload  --repo codespell --tag v$(VERSION) \
-		--name codespell-$(VERSION).tar.xz.asc \
-		--file codespell-$(VERSION).tar.xz.asc
-
```

Comment:
IIRC I just tagged a release on the GitHub GUI, then did `python setup.py sdist upload`